internal/controller/dnshealthcheckprobe_reconciler.go

```diff
@@ -40,6 +40,7 @@ type DNSProbeReconciler struct {
 //+kubebuilder:rbac:groups=kuadrant.io,resources=dnshealthcheckprobes,verbs=get;list;watch;create;update;patch;delete
 //+kubebuilder:rbac:groups=kuadrant.io,resources=dnshealthcheckprobes/status,verbs=get;update;patch
 //+kubebuilder:rbac:groups=kuadrant.io,resources=dnshealthcheckprobes/finalizers,verbs=update
+//+kubebuilder:rbac:groups="",resources=configmaps,verbs=get;list;watch;create;update;patch;delete
```

Comment:
Do you really need all these permissions, also see above https://github.com/Kuadrant/dns-operator/pull/283/files#r1827561916